src/Backend/EdgeTPU/EdgeTPUToolchain.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from "../Command";
+import { DebianToolchain } from "../ToolchainImpl/DebianToolchain";
+
+import * as ini from "ini";
+import * as fs from "fs";
+import * as path from "path";
+
+class EdgeTPUDebianToolchain extends DebianToolchain {
+  run(cfg: string): Command {
+    let cmd = new Command("edgetpu_compiler");
+    var config = ini.parse(fs.readFileSync(cfg, "utf-8").trim());
+
+    if (config["one-import-edgetpu"] === undefined) {
+      return cmd;
+    }
```

Comment:
When there is not `one-import-edgetpu` section in a edgetpu cfg file, it will return `edgetpu_compiler` as command and run it. Then it will print out a help manual like below.

```
edgetpu_compiler                                                                                                                                                    ✔  10120  15:43:56
Edge TPU Compiler version 16.0.384591198

Usage:
edgetpu_compiler [options] model...

Options:
-o, --out_dir <dir>
       Output the compiled model and log files to directory <dir>.
       Default is the current directory (.)
-m, --min_runtime_version <val>
       Minimum Edge TPU runtime version required by your model.
       Models are forward-compatible with new Edge TPU runtimes.
       Default: 14; Min: 10; Max: 14
-i, --intermediate_tensors <str>
        A comma-separated list of tensors you want as outputs from
        the Edge TPU custom op. All operators following these tensors
       will not be compiled for the Edge TPU but are still in the
       .tflite file and will run on the CPU. If you have multiple
       input models (for co-compilation), then separate the list of
       tensors for each model with a colon, and pass them in the
       order corresponding to the input models.
-n, --num_segments <val>
       Partition the model into [num_segments] segments.
       Default value is 1 (no partition).
-s, --show_operations
       Print the log showing operations that mapped to the Edge TPU.
-a, --enable_multiple_subgraphs
       Enable multiple(all) subgraphs (experimental flag).
-d, --search_delegate
       Enable repeated search for an Edge TPU delegate.
-k, --delegate_search_step <val>
       Custom step size for search_delegate.
       Default value is 1.
-t, --timeout_sec <val>
       Timeout for one compilation process in seconds.
       Default value is 180.
-v, --version
       Print compiler version and exit.
-h, --help
       Print this help and exit.

No input model(s).
```

How about putting a log or a message like `Fail to run. The configuration file doesn't include `one-import-edgetpu` section. Please add one.`?

Reply:
Thanks for the suggestion!
We will put that log or message.